tasks/23_seek-destroy/seek-destroy.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable prefer-rest-params */
+
+// Arrow functions don't have an access to arguments object
+// so classic function must be used.
+// First, create an empty array and copy arguments with call method,
+// then filter out arguments.
+
+export function destroyer( arr ) {
+  const args = Array.prototype.slice.call( arguments );
+  arr = arr.filter( (item) => args.indexOf( item)==-1 ? true :false );
```

Comment:
it is better to use `Array.prototype.includes(item)` in this case you will not need ternary operator.

Reply:
I don't understand how I could implement Array.includes(item). Also when I use `...rest` tests fail. Probably I'm doing something wrong. I need a help here.